src/Auth0.ManagementApi/Paging/PaginationInfo.cs

```diff
@@ -11,26 +11,40 @@ public class PaginationInfo
         /// <param name="pageNo">Page index of the results to return. First page is 0.</param>
         /// <param name="perPage">Number of results per page.</param>
         /// <param name="includeTotals">Whether to return the complete total result count (true) or not (false).</param>
-        public PaginationInfo(int pageNo = 0, int perPage = 50, bool includeTotals = false)
+        /// <param name="from">When using checkpoint pagination, an Id from which to begin the next page of results.</param>
+        /// <param name="take">When using checkpoint pagination, the number of results per page.</param>
+        public PaginationInfo(int? pageNo = 0, int? perPage = 50, bool? includeTotals = false, string from = null, int? take = null)
```

Comment:
That's a much cleaner route to go, thanks. I forget overloaded methods are even an option, since I mostly work in languages where that doesn't exist. I'm working on updating the PR to use this approach.